### PR TITLE
Merge our default develop branch

### PR DESCRIPTION
**Pull request by vladanvi99**
# JavaScript capstone project - Your API-based webapp

In this JavaScript capstone project is about building our web application based on an external API.
- It fetches movies from an API and displays them
- It allows like a movie.
- It allows to view details and add comments

To watch a video explaining in more details what we did, [click here](https://youtu.be/2h8Ad4uiYHU)

In this pull request, we want to merge our default development branch with the main branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+- Github: [@mugberto](https://github.com/mugberto)
+- Twitter: [@mugberto](https://twitter.com/mugberto)
+- Linkedin: [mugberto](https://www.linkedin.com/in/hubert-mugabo-23144b6a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```
Comment (henatan99) [inline README.md:60]: - [ ] Please, acknowledge any person or party whom resources are being used or referenced in your project.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "webpack-boilerplate",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "jest",
```
Comment (henatan99) [inline package.json:7]: - [ ] Please, do some workaround with your `test` config as it is complaining on my end and I cannot run the test. 
Screenshot from 2021-08-07 00-59-55

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @vladanvi99 && @mugberto , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- Honestly, you guys did a hell of a job. Very organized and clean code. Slick on the user end. I commend you!!! :clap: :clap: :clap: 

## Required Changes ♻️
_Check the comments under the review._

- [ ] All the members in the video should appear in the video and talk about the project. kindly record your video again. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mugberto** [comment]
Thank you @henatan99 for the review. 
We have made another video as per request. We ran tests on two different operating systems and everything went fine. Maybe some of the dependencies were installed correctly. You can run again `npm install`

**Review by bellom — APPROVED** [review]
## Hi @vladanvi99 @mugberto  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**